### PR TITLE
CDK Istio readme

### PR DESCRIPTION
**Pull request by preddy727**
*Issue #, if available:*

*Description of changes:*

CDK Istio readme with bookinfo example and visualization 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+# Single Cluster Open Source Observability - Istio Monitoring
+
+## Objective
+
+This pattern demonstrates how to use the _New EKS Cluster Open Source Observability Accelerator_ with Istio based workloads.
+
+It also enables control plane logging for comprehensive monitoring on the EKS cluster.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:7]: This line can be removed.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
+# Single Cluster Open Source Observability - Istio Monitoring
+
+## Objective
+
+This pattern demonstrates how to use the _New EKS Cluster Open Source Observability Accelerator_ with Istio based workloads.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:5]: Can you talk a little bit about the background what is istio, what is it for monitoring istio and what we are trying to accomplish. You can refer our blogs.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+    "fluxRepository": {
+      "name": "grafana-dashboards",
+      "namespace": "grafana-operator",
+      "repository": {
+        "repoUrl": "https://github.com/aws-observability/aws-observability-accelerator",
+        "name": "grafana-dashboards",
+        "targetRevision": "main",
+        "path": "./artifacts/grafana-operator-manifests/eks/infrastructure"
+      },
+      "values": {
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:33]: Add the missing DASH URLs mandatory as discussed.

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+
+Once completed the rest of the _Deploying_ steps, you can move on with the deployment of the Istio workload.
+
+## Visualization
+
+### 1. Grafana dashboards
+
+Go to the Dashboards panel of your Grafana workspace. You will see a list of Istio dashboards under the `Observability Accelerator Dashboards`
+
+<img width="1208" alt="image" src="https://github.com/aws-observability/terraform-aws-observability-accelerator/assets/34757337/19b589b4-00f6-465d-a562-1da39e8b9b8c">
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:60]: Why is this referring to terraform snapshot? Can we have one created in images?

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+
+Open one of the Istio dasbhoards and you will be able to view its visualization
+
+<img width="1850" alt="image" src="https://user-images.githubusercontent.com/47993564/236842708-72225322-4f97-44cc-aac0-40a3356e50c6.jpeg">
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:64]: Can we add these images to this doc images folder and refer it,

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+### 2. Amazon Managed Service for Prometheus rules and alerts
+
+Open the Amazon Managed Service for Prometheus console and view the details of your workspace. Under the `Rules management` tab, you will find new rules deployed.
+
+<img width="1054" alt="image" src="https://user-images.githubusercontent.com/47993564/236844084-80c754e3-4fe1-45bb-8361-181432675469.jpeg">
+
+!!! note
+    To setup your alert receiver, with Amazon SNS, follow [this documentation](https://docs.aws.amazon.com/prometheus/latest/userguide/AMP-alertmanager-receiver.html)
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:73]: Same feedback on screenshot

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 7]
```diff
@@ -0,0 +1,129 @@
...
+
+For the Bookinfo sample application, visit `http://$GATEWAY_URL/productpage` in your web browser. To see trace data, you must send requests to your service. The number of requests depends on Istio’s sampling rate and can be configured using the Telemetry API. With the default sampling rate of 1%, you need to send at least 100 requests before the first trace is visible. To send a 100 requests to the productpage service, use the following command:
+```sh
+for i in $(seq 1 100); do curl -s -o /dev/null "http://$GATEWAY_URL/productpage"; done
+```
+
+### 3. Explore the Istio dashboards
+
+Log back into your Amazon Managed Grafana workspace and navigate to the dashboard side panel. Click on the `Observability Accelerator Dashboards` folder and open the `Istio Service` Dashboard. Use the Service dropdown menu to select the `reviews.default.svc.cluster.local` service. This gives details about metrics for the service, client workloads (workloads that are calling this service), and service workloads (workloads that are providing this service).
+
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:120]: Could we should screenshot of the Service Dashboard

File: docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md [hunk 8]
```diff
@@ -0,0 +1,129 @@
...
+Explore the Istio Control Plane, Mesh, and Performance dashboards as well.
```
Comment (elamaran11) [inline docs/patterns/single-new-eks-observability-accelerators/single-new-eks-istio-opensource-observability.md:121]: Also can you please add screenshots for all the 4 dashboards so people can see it.

**Review by elamaran11 — CHANGES_REQUESTED** [review]
You need to chain this doc to `mkdocs.yaml` file so the doc goes to public website.

**Comment by preddy727** [comment]
> You need to chain this doc to `mkdocs.yaml` file so the doc goes to public website.

Mkdocs.yaml link updated.

**Pull request closed without merging**